README.md

```diff
@@ -1,24 +1,35 @@
 # FilePathRouter
-The Blank Framework FilePathRouter is an extremely simple router library for routing requests. Essentially all it does
-is take a path like /blog/posts and routes it to a directory like /blog/posts/index.php.
 
-To use it you need to simply instantiate the class and pass the **full** path into the constructor to the directory
-where your routes will be. An example as from the this project might be assuming your index.php is where the README is
+The Blank Framework FilePathRouter is an efficient and straightforward router library designed for routing HTTP requests
+in PHP applications. It adeptly maps a requested path, such as `/blog/posts`, to its corresponding directory and file,
+e.g., `/blog/posts/index.php`.
+
+## Usage
+
+To utilise this library, instantiate the `FilePathRouter` class and pass the **absolute** path to the directory that
+contains your routes. Here's an example that assumes your `index.php` is located in the same directory as this README:
 
 ```php
 $router = new FilePathRouter(__DIR__ . '/routes');
 ```
 
```

Comment:
To route your requests, get the requested path for example `/blog` and pass it into the function `routeRequest` as shown below:

**INSERT CODE SNIPPET HERE THAT WAS REMOVED**